sources/niLang/src/System_Linux.cpp

```diff
@@ -971,31 +990,140 @@ class cLinuxWindow : public ni::ImplRC<ni::iOSWindow,ni::eImplFlags_Default,ni::
     return false;
   }
 
-  ///////////////////////////////////////////////
+  // This function will get the monitor which has more overlapping
+  // area with the app window.
```

Comment:
We might need this for multiple monitor support. I only have a monitor right now so I couldn't test this for real other than verifying that works with one monitor :P